label_studio/data_manager/managers.py

```diff
@@ -498,8 +513,56 @@ def __init__(self, expression, distinct=False, output_field=None, **extra):
 def annotate_completed_at(queryset):
     from tasks.models import Annotation
 
+    LseProject = load_func(settings.LSE_PROJECT)
+    get_tasks_agreement_queryset = load_func(settings.GET_TASKS_AGREEMENT_QUERYSET)
+
```

Comment:
Let's do an early return here. Easy to read:
```
    if not (get_tasks_agreement_queryset and LseProject and queryset.exists()):
        return queryset
```

Reply:
Btw not sure I understand this?
`LseProject.objects.filter(project_id=queryset[0].project_id).exists()`
Should not be enought with `queryset.exists())` os is a chance that this project_id doesn't exit?